components/table/test/table.sticky.visual-diff.html

```diff
@@ -1,384 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Was this file deletion just missed in a previous refactor?

Reply:
Yeah I definitely deleted it, but it may have snuck back in by accident as a conflict in one of the many rebases I've had to do to keep all these branches in sync.